src/middlewared/middlewared/plugins/network.py

```diff
@@ -607,8 +608,12 @@ def iface_extend(self, iface_state, configs, is_freenas, fake=False):
         """
         Returns wether or not we are waiting user to checkin the applied network changes
         before they are rolled back.
+        Value is in number of seconds or null.
         """
-        return self._rollback_timer is not None
+        if self._rollback_timer:
+            remaining = self._rollback_timer.when() - time.monotonic()
```

Comment:
We should use `loop.time()` here:

```
when()
Return a scheduled callback time as float seconds.

The time is an absolute timestamp, using the same time reference as loop.time().
```

Reply:
Which is just a wrapper for time.monotonic ?
Or you just want to be safe for future changes to another event loop?

Reply:
Why depend on implementation if we can avoid it easily?

I did not know that `loop.time()` is a wrapper for `time.monotonic()`, my first thought was "it this a bug?"